### PR TITLE
Small translation changes 

### PR DESCRIPTION
**Pull request by b1rdhous3**
**1.** Tell us something about your changes.
Fixed typo, deleted unneeded stuff and added few translations.

**2.** If this changes affect the .conf file. Please provide the changed section

**3.** Reference an issue (optional)

Note: For further discussions use our forum: forum.hyperion-project.org

### CODE REVIEW

File: assets/webconfig/i18n/en.json [hunk 1]
```diff
@@ -365,7 +363,7 @@
 		"edt_append_hz" : "Hz",
 		"edt_append_pixel" : "Pixel",
 		"edt_append_percent" : "%",
-		"edt_append_degree" : "�",
+		"edt_append_degree" : "�",
```
Comment (brindosch) [inline assets/webconfig/i18n/en.json]: Hm? That worked for me, i see now just the black squared question mark.

Comment (penfold42) [inline assets/webconfig/i18n/en.json]: &deg; ?

**Comment by b1rdhous3** [comment]
Now?

File: assets/webconfig/i18n/en.json [hunk 2]
```diff
@@ -158,7 +156,7 @@
 		"support_label_yttext" : "Bored from pictures? Checkout our Youtube channel!",
 		"support_label_donate" : "Donate or use our affiliate links",
 		"support_label_affinstr1" : "Click on the appropriate link of your country",
-		"support_label_affinstr2" : "Everything you buy (doesn�t matter what) we get a small fee based on your turnover",
+		"support_label_affinstr2" : "Everything you buy (doesn�t matter what) we get a small fee based on your turnover",
```
Comment (brindosch) [inline assets/webconfig/i18n/en.json]: doesn�t
please too :)

**Comment by brindosch** [comment]
I noticed my local copy of en.json is ANSI. My fault seems so

**Comment by b1rdhous3** [comment]
Ah okay, there we go. :)

File: assets/webconfig/i18n/en.json (hunk 1 above)
Comment (b1rdhous3) [inline assets/webconfig/i18n/en.json]: @penfold42 I think we should face the encoding problems instead of that ampersand "old way" :)

**Comment by redPanther** [comment]
have assets/webconfig/i18n/en.json now right encoding (utf8)?

**Comment by brindosch** [comment]
yes

**Comment by redPanther** [comment]
👍

**Pull request merged**